app/src/main/java/com/example/parkingsystem/mvp/presenter/ConfigureParkingDialogPresenter.java

```diff
@@ -0,0 +1,31 @@
+package com.example.parkingsystem.mvp.presenter;
+
+import com.example.parkingsystem.listener.ConfigureParkingDialogListener;
+import com.example.parkingsystem.mvp.contracts.ConfigureParkingDialogContract;
+
+public class ConfigureParkingDialogPresenter implements ConfigureParkingDialogContract.ConfigureParkingDialogPresenter {
+    private ConfigureParkingDialogContract.ConfigureParkingDialogView view;
+    private ConfigureParkingDialogListener listener;
```

Comment:
the listener should not be an instance variable.. you should pass it when it is needed, no in the presenter consturctor

Reply:
I see, I only need the listener when I press the confirm button. Changed!